docs/introduction/start-building-your-application.md

```diff
@@ -0,0 +1,114 @@
+
+*Note: When you change the domain settings, it is likely that [tests](/docs/introduction/automated-testing.md) (mainly [acceptance](/docs/introduction/automated-testing.md#acceptance-tests-aka-functional-tests-or-selenium-tests)) will fail and that [demo data](/docs/introduction/basic-and-demo-data-during-application-installation.md) won't fit you.
+It's up to you to adjust them according to your needs.*
+
+## What is a locale and how to set up locales
+
+A locale is a combination of language and national settings like a collation of language-specific characters.
+We use [ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) to specify locale *(eg. `cs`, `de`, `en`, `sk`)*.
+Every domain has defined one locale and also administration has defined it's locale.
+
+When you install the project, locales are
+* `shopsys` *(1st domain)*: `en`
+* `2.shopsys` *(2nd domain)*: `cs`
+* administration: `en`
+
+Please read [how to change locale settings](/docs/introduction/how-to-set-up-domains-and-locales.md#3-locale-settings) if you need to change the locale setting of a domain or specifically [administration locale](/docs/introduction/how-to-set-up-domains-and-locales.md#36-locale-in-administration).
+
+## Where is Elasticsearch used and how to set it up
+
+We use Elasticsearch on the frontend for product searching, filtering and for fast listing of products and it provides great performance.
+
+*Please find more in [detailed article](/docs/model/elasticsearch.md) about Elasticsearch usage.*
+
+Every domain have defined one [index](/docs/model/elasticsearch.md#elasticsearch-index-setting), you can find and adjust the index structure in files `src/Shopsys/ShopBundle/Resources/definition/<domain_id>.json`.
+The most often change is adding [fields](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html) and changing [analysis](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis.html) to justify search behavior.
+
+Data are not immediately available in Elasticsearch after import or change, they have to be [exported](/docs/introduction/console-commands-for-application-management-phing-targets.md#product-search-export-products) first.
+
+## How to set up routing
+
+Routing is a mechanism that maps URL path to controllers and methods.
+
+*We use Symfony routing, so please find more in the [official documentation](https://symfony.com/doc/3.4/routing.html)*
+
+You can adjust the routing in configuration `src/Shopsys/ShopBundle/Resources/config/routing_front.yml` and [locale-specific](/docs/introduction/how-to-set-up-domains-and-locales.md#32-frontend-routes) in configuration `src/Shopsys/ShopBundle/Resources/config/routing_front_xx.yml`.
+
+## What is a default currency and how to set it up
+
+A default currency is a currency that is displayed when showing a price and no currency is specified explicitly.
```

Comment:
This sentence is confusing, I do not know what it says.

Reply:
better?